src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const populationList = document.querySelectorAll('.population');
+const populationArr = [...populationList].map(el =>
+  +(el.innerText.split(',').join('')));
+const total = populationArr.reduce((a, b) => (a + b));
+const average = total / populationArr.length;
+
+document.querySelector('.total-population').textContent
+  = total.toLocaleString();
```

Comment:
Do not forget to set format you need
![2023-05-15_16h30_16](https://github.com/mate-academy/js_get_data_DOM/assets/61904995/e3f6673b-c568-497a-976a-820d613e7e6f)


```suggestion
  = total.toLocaleString('en-US');
```